modules/Bio/EnsEMBL/Variation/VariationFeature.pm

```diff
@@ -1968,7 +1967,117 @@ sub hgvs_genomic {
 
 }
 
+=head2  spdi_genomic
+
+Arg [1]     : int (Optional)
+              It also returns a SPDI notation for the reference allele. By default value is '0'.
+              '1' -> returns a SPDI for the reference allele;
+              '0' -> doesn't return a SPDI for the reference allele;
+Example     : my $vf = $variation_feature_adaptor->fetch_by_dbID(rs145160881);
```

Comment:
parameter to fetch_by_dbID should be a variation_feature_id or a  "location_identifier"
Suggest say "For example" instead of will print